libkbfs/block_retrieval_queue.go

```diff
@@ -351,29 +351,31 @@ func (brq *blockRetrievalQueue) Request(ctx context.Context,
 			}
 		}
 		br.reqMtx.Lock()
+		defer br.reqMtx.Unlock()
```

Comment:
I needed to move this so that the assignment of `br.priority` is protected by the lock.

Reply:
It's a bit confusing that this is in a `for` loop, though it seems kosher since we never `continue` after this, only `return`.  Still, it might be more clear to define `br` outside of the `for` loop, then break it after the `if` statement above, so that the locking happens after the loop.